.circleci/template.yml

```diff
@@ -0,0 +1,273 @@
+version: 2.1
+
+parameters:
+  triggerCommitHash:
+    type: string
+    default: ""
+  isNightly:
+    type: boolean
+    default: false
+  isRelease:
+    type: boolean
+    default: false
+
+commands:
+  bootstrap_repository_command:
+    description: "Bootstrap the repository"
+    steps:
+      - install_ssh_keys_command
+      - run:
+          name: Install dependencies
+          command: yarn install
+      - prepare_environment_variables_commands
+
+  install_ssh_keys_command:
+    description: "Install SSH keys"
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "a0:41:a2:56:c8:7d:3f:29:41:d1:87:92:fd:50:2b:6b"
+
+  npm_login_command:
+    description: "Enable interacting with `npm` using an auth token"
+    steps:
+      - run:
+          name: Login to the npm registry using '.npmrc' file
+          command: echo "//registry.npmjs.org/:_authToken=\${CKE5_NPM_TOKEN}" > ~/.npmrc
+
+  git_credentials_command:
+    description: "Setup git configuration"
+    steps:
+      - run:
+          name: Setup git configuration
+          command: |
+            git config --global user.email "ckeditor-bot@cksource.com"
+            git config --global user.name "CKEditorBot"
+
+  prepare_environment_variables_commands:
+    description: "Prepare non-secret environment variables"
+    steps:
+      - run:
+          name: Prepare environment variables
+          command: |
+            #!/bin/bash
+
+            # Non-secret environment variables needed for the pipeline scripts.
+            CKE5_GITHUB_ORGANIZATION="ckeditor"
+            CKE5_GITHUB_REPOSITORY="ckeditor5-dev"
+            CKE5_CIRCLE_APPROVAL_JOB_NAME="release_approval"
+            # CKE5_GITHUB_RELEASE_BRANCH="master"
+            CKE5_GITHUB_RELEASE_BRANCH="epic/3757-esm"
```

Comment:
This change must be updated to `#master` before merging. The entire commit 1cc2de522b9771463d32f9453f18814f87fbca24 should be reverted.

Reply:
Done in 2722abce9c964cc1a800af51ab5cd54510db12ab.